docs/src/debugging/function-selector.md

```diff
@@ -0,0 +1,18 @@
+# Function selector
```

Comment:
This file can use some Markdown linting.

Reply:
I've ran it through the default settings of [mdl](https://github.com/markdownlint/markdownlint). Suggested the empty lines around the code block.

Also complained about the line width. But I guess we would need to settle on a linter and the settings for it.

Reply:
I usually use https://github.com/DavidAnson/markdownlint which has a VS code plugin. Wouldn't worry much about line width but vertical spaces are nice to have.